gym/envs/registration.py

```diff
@@ -547,7 +547,7 @@ def make(
     id: Union[str, EnvSpec],
     max_episode_steps: Optional[int] = None,
     autoreset: bool = False,
-    new_step_api: bool = False,
+    new_step_api: bool = True,
```

Comment:
Rename to 'apply_step_compatibility' as in a few years time new or old won't be descriptive